05-geometry-operations.Rmd

```diff
@@ -559,7 +559,7 @@ A variety of georeferencing techniques exist, including:
 
 - Georectification based on known [ground control points](https://www.qgistutorials.com/en/docs/3/georeferencing_basics.html);
 - Orthorectification, which also accounts for local topography
-- Image [registration](https://en.wikipedia.org/wiki/Image_registration) is used to combine images of the same thing but shot from different sensors the process of aligning one image with another (in terms of coordinate system, and resolution)
+- Image [registration](https://en.wikipedia.org/wiki/Image_registration) is used to combine images of the same thing but shot from different sensors by aligning one image with another (in terms of coordinate system, and resolution)
```

Comment:
Definite improvement. While reading this `, and` jumps out at me and I suggest removing the superfluous punctuation, e.g. updating this to read:

```
- Image [registration](https://en.wikipedia.org/wiki/Image_registration) is used to combine images of the same thing but shot from different sensors by aligning one image with another (in terms of coordinate system and resolution)
```

Sound reasonable?

Reply:
Absolutely! I'll update it :)